### ISSUE TITLE
Installing Globalize 5.1.0 with Rails 5.2 gives error on bundle install.

### ISSUE THREAD

**Issue by jkepps** [globalize/globalize#735]
I'm using bundler to install globalize and my lock file is specifying version 5.1.0.
```
    globalize (5.1.0)
      activemodel (>= 4.2, < 5.3)
      activerecord (>= 4.2, < 5.3)
      request_store (~> 1.0)
```

when running `bundle install` i get the following error:
```
Fetching globalize 5.1.0
Downloading globalize-5.1.0 revealed dependencies not in the API or the lockfile (activerecord (>= 4.2, <
5.2), activemodel (>= 4.2, < 5.2)).
Either installing with `--full-index` or running `bundle update globalize` should fix the problem.
```

**Comment by michalzynek** [comment]
Got similar error when added globalize, rails 5.2.3, ruby 2.6.1
```
Bundler could not find compatible versions for gem "activemodel":
  In snapshot (Gemfile.lock):
    activemodel (= 5.2.3)

  In Gemfile:
    active_model_serializers was resolved to 0.10.10, which depends on
      activemodel (>= 4.1, < 6.1)

    globalize (~> 5.1.0) was resolved to 5.1.0, which depends on
      activemodel (>= 4.2, < 5.2)

    rails (~> 5.2.3) was resolved to 5.2.3, which depends on
      activemodel (= 5.2.3)

    rspec-activemodel-mocks was resolved to 1.1.0, which depends on
      activemodel (>= 3.0)

    web-console (>= 3.3.0) was resolved to 3.7.0, which depends on
      activemodel (>= 5.0)
```

**Comment by raghulrnair** [comment]
Got error when adding globalize to  Rails 6.0.1, ruby 2.5.5

```

Bundler could not find compatible versions for gem "activemodel":
  In snapshot (Gemfile.lock):
    activemodel (= 6.0.1)

  In Gemfile:
    globalize (~> 5.2.0) was resolved to 5.2.0, which depends on
      activemodel (< 5.3, >= 4.2)

    rails (~> 6.0.0) was resolved to 6.0.1, which depends on
      activemodel (= 6.0.1)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.

```

**Comment by parndt** [comment]
You need version `5.3.0` for Rails 6.0.x support
